docs/user_guides/tls_proxy.md

```diff
@@ -1,12 +1,12 @@
 # How to enable use TLS proxy
 
-Milvus proxy uses TLS mutual authentication. 
+Milvus proxy uses TLS mutual authentication.
```

Comment:
Do we need to mention that Milvus supports both client-to-server and server-to-server TLS here?

Reply:
No, because the server-to-server can be also seen as the client-to-server.